### ISSUE TITLE
Training LJSpeech as described in the README.md

### ISSUE THREAD

**Issue by SamuelLarkin** [ming024/FastSpeech2#47]
Hi,
  I'm trying to run the LJSpeech example described in the `README.md` but it fails during preprocessing.
Here's what I've done so far.

```bash
git clone https://github.com/ming024/FastSpeech2.git
cd FastSpeech2
```

I then downloaded `LJSpeech.zip` from https://drive.google.com/drive/folders/1DBRkALpPd6FL9gjHMmMEdHODmkgNIIK4?usp=sharing

Looking at what is in `LJSpeech.zip`

```bash
unzip -l LJSpeech.zip | head
Archive:  LJSpeech.zip
  Length      Date    Time    Name
---------  ---------- -----   ----
        0  2021-02-13 20:13   TextGrid/
        0  2021-02-13 20:13   TextGrid/LJSpeech/
     7164  2021-02-13 20:06   TextGrid/LJSpeech/LJ001-0019.TextGrid
     4589  2021-02-13 20:06   TextGrid/LJSpeech/LJ001-0020.TextGrid
     3830  2021-02-13 20:06   TextGrid/LJSpeech/LJ001-0061.TextGrid
     8199  2021-02-13 20:06   TextGrid/LJSpeech/LJ001-0088.TextGrid
     8678  2021-02-13 20:06   TextGrid/LJSpeech/LJ001-0089.TextGrid
	 ....
     9461  2021-02-13 20:06   TextGrid/LJSpeech/LJ049-0158.TextGrid
     3698  2021-02-13 20:06   TextGrid/LJSpeech/LJ050-0230.TextGrid
     8067  2021-02-13 20:06   TextGrid/LJSpeech/LJ050-0261.TextGrid
     7719  2021-02-13 20:06   TextGrid/LJSpeech/LJ050-0272.TextGrid
---------                     -------
 87949622                     13102 files

Then the instructions say to unzip it `unzip -d preprocessed_data/LJSpeech/TextGrid/ LJSpeech.zip`.

Looking at the result from the previous command.

```bash
find preprocessed_data/LJSpeech/TextGrid
preprocessed_data/LJSpeech/TextGrid
preprocessed_data/LJSpeech/TextGrid/LJSpeech
preprocessed_data/LJSpeech/TextGrid/LJSpeech/LJ003-0286.TextGrid
preprocessed_data/LJSpeech/TextGrid/LJSpeech/LJ008-0301.TextGrid
preprocessed_data/LJSpeech/TextGrid/LJSpeech/LJ002-0235.TextGrid
....
```

I then symlink my LJSpeech wav files under `raw_data`

```bash
( cd raw_data && ln -fs  <base_dir>/LJ.Speech.Dataset/LJSpeech-1.1/wavs   LJSpeech; )
```

I then modified `preprocess.yaml` to match my paths.

`cp config/LJSpeech/preprocess.yaml preprocess.yaml`

```diff
4c4
<   corpus_path: "/home/ming/Data/LJSpeech-1.1"
---
>   corpus_path: "<base_dir>/LJ.Speech.Dataset/LJSpeech-1.1"
6c6,7
<   raw_path: "./raw_data/LJSpeech"
---
>   #raw_path: "./raw_data/LJSpeech"
>   raw_path: "./raw_data"
```

And when I try to run `python3 preprocess.py preprocess.yaml` I get the following error for missing `lab` files

```bash
Processing Data ...
  0%|                                                                                                                                                  | 0/1 [00:00<?, ?it/s]
Traceback (most recent call last):
  File "preprocess.py", line 15, in <module>
    preprocessor.build_from_path()
  File "/gpfs/fs1/nrc/ict/others/u/sam037/tts/FastSpeech2.git/preprocessor/preprocessor.py", line 79, in build_from_path
    ret = self.process_utterance(speaker, basename)
  File "/gpfs/fs1/nrc/ict/others/u/sam037/tts/FastSpeech2.git/preprocessor/preprocessor.py", line 179, in process_utterance
    with open(text_path, "r") as f:
FileNotFoundError: [Errno 2] No such file or directory: './raw_data/LJSpeech/LJ026-0087.lab'
```

What steps am I missing?

**Comment by 1653134** [comment]
You should follow this path and check file "LJ026-0087.lab". Maybe this file don't exist or the path is wrong.

**Comment by SamuelLarkin** [comment]
Hi,
  * There are no lab files present when we clone this repo.
```bash
git clone https://github.com/ming024/FastSpeech2.git
find FastSpeech2 -name \*.lab | wc -l
0
```
  * There are no lab files in `LJSpeech.zip` that you provide.
```bash
unzip -l LJSpeech.zip | grep -c lab
0
```
  * There are no lab files in `LJSpeech-1.1.tar.bz2` 
```bash
tar tf LJSpeech-1.1.tar.bz2 | grep -v 'wav$'
LJSpeech-1.1/
LJSpeech-1.1/metadata.csv
LJSpeech-1.1/wavs/
LJSpeech-1.1/README
```

My guess is that the lab files are the content of `metadata.csv` spread across 13,100 files instead of been more efficiently kept in `metadata.csv`.

Is the content of a lab file the transcription or the normalized transcription from `metadata.csv`?
Knowing which one they should contain, I will write a small script to generate those from `metadata.csv`.

Thanks

**Comment by ming024** [comment]
@SamuelLarkin you should run ``prepard_align.py`` first. Find the details in ``README.md``.

**Comment by ming024** [comment]
closed #47
